lib/scripts/controllers/options.js

```diff
@@ -285,6 +293,18 @@ export class OptionCtrl {
     }
   }
 
+  updatePlaybackOptions(){
+    var playbackOpts = JSON.stringify({
+      'volume': this.volume,
+      'rate': this.rate,
+      'voiceName': this.voiceName,
+      'pitch': this.pitch
+    });
+    this.save({
```

Comment:
@rohankatyal29 wouldn't this show the message a lot of times? Since you are doing an ng-change.